core/ts.core/build.properties

```diff
@@ -1,5 +0,0 @@
-source.. = src/

```

Comment:
why did this file and the following disappeared :/ 

Could you please keep only meaningful changes please?

Reply:
Oups, nevermind :)

Reply:
I changed target branch and there are a lot less changes :) (It now points to mavenPackaging 11 branch)